src/main/java/edu/greenblitz/robotName/subsystems/swerve/Chassis/ChassisConstants.java

```diff
@@ -32,6 +35,10 @@ public class ChassisConstants {
     public static final double DRIVER_LINEAR_SPEED_FACTOR_SLOW = 0.5;
 
     public static final boolean ANGULAR_JOYSTICK_INVERTED = true;
+
+    public static final boolean LINEAR_JOYSTICK_INVERTED = false;
+    public static final MoveByJoysticks.DriveMode CURRENT_DRIVE_MODE = MoveByJoysticks.DriveMode.NORMAL;
```

Comment:
Why is `CURRENT_DRIVE_MODE` final?

The name implies that this value changes throughout the runtime